code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -3,6 +3,7 @@
 	desc = "To stop that awful noise."
 	icon_state = "muzzle"
 	inhand_icon_state = "blindfold"
+	clothing_flags = BLOCKS_SPEECH // I'd recommend a double-check on this flag before it goes live.
```

Comment:
This comment isn't necessary, what you put in code should be what is able to go into production--comments like this should be best left in a comment on the PR.

Reply:
```suggestion
	clothing_flags = BLOCKS_SPEECH
```